### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by brahimdidi**
In this project I will be reviewing my own code.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  <form id="survey-form">
+    <label id="name-label">Name</label>
+    <input id="name" type="text" required placeholder="input your name">
+
+    <label id="email-label">Email</label>
+    <input id="email" type="email" required placeholder="input your email">
+
+    <label id="number-label">Age</label>
+    <input id="number" type="number" required max="70" min="18" placeholder="age">
+    <select id="dropdown">
```
Comment (brahimdidi) [inline index.html:26]: Too many ids are used, please try to use fewer ids and more classes.
classes make your code clean and easy to read

File: index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    <input type="checkbox" id="" name="Team" value="Barcelona">
+    <label for="female">Female</label><br>
+    <input type="checkbox" id="" name="Team" value="Real Madrid">
+    <label for="other">Other</label>
+  </p>
+  <p><textarea placeholder="additional comments"></textarea></p>
+  <p><input type="submit" id="submit" value="submit"></p>
+  </form>
+
+
```
Comment (brahimdidi) [inline index.html:52]: Please add inline comments to your HTML. comments make your code more structured and easier to read

**Review by brahimdidi — COMMENTED** [review]
## Changes required :recycle: 

Please check my comments below and make the required changes.

Happy coding.

Thanks :grin:

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**